arcgis-ios-sdk-samples/Content Display Logic/Views/GroupLayersSectionView.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+final class GroupLayersSectionView: UITableViewHeaderFooterView {
+    @IBOutlet var layerNameLabel: UILabel!
+    @IBOutlet var layerVisibilitySwitch: UISwitch?
+    weak var delegate: GroupLayersSectionViewDelegate?
+    
+    static let reuseIdentifier = String(describing: self)
+    
+    static var nib: UINib {
```

Comment:
I know it's been said that a `UITableViewHeaderFooterView` can be created from a XIB. After all, Apple has an API for registering a header/footer view with one. But one thing that's always bothered me about that approach is that it goes against Apple's own documentation. What do I mean? Take the following:

> If you have custom content to display, create the subviews for your content and add them to the view in the `contentView` property.

That's not where the custom content ends up when it comes out of the XIB. Instead of being subviews of the content view, they end up as subviews of the header/footer view itself:

![Screen Shot 2019-04-11 at 3 51 43 PM](https://user-images.githubusercontent.com/2257493/56002380-08cc9680-5c77-11e9-9c5d-bf7595ba19ba.png)

That may be perfectly fine in practice. I don't know. Just something to keep in mind.